contentcuration/contentcuration/static/js/edit_channel/models.js

```diff
@@ -295,6 +295,20 @@ var ContentNodeModel = BaseModel.extend({
             });
         });
     },
+    fetch_details: function() {
```

Comment:
API call to get all information from a topic, for now, just getting details from root node. Later, this will get it for topics at a lower level